synapse/models/person.py

```diff
@@ -490,13 +490,24 @@ def getModelDefs(self):
                     ('crypto:address', ('crypto:currency:address', {}), {
                         'doc': 'A crypto currency address associated with the contact.'
                     }),
-
                     ('lang', ('lang:language', {}), {
                         'alts': ('langs',),
                         'doc': 'The language specified for the contact.'}),
-
                     ('langs', ('array', {'type': 'lang:language'}), {
                         'doc': 'An array of alternative languages specified for the contact.'}),
+                    ('banner', ('file:bytes', {}), {
+                        'doc': 'The file representing the banner for the contact.'
```

Comment:
the typical model layout puts the closing `}),` for properties on the line with the `doc` string and leaves one blank line :) i've been making it more consistent in the 3x branch ( not that this will reach there... )